sampleconfig/core.yaml

```diff
@@ -659,6 +659,14 @@ chaincode:
     # Format for the chaincode container logs
     format: "%{color}%{time:2006-01-02 15:04:05.000 MST} [%{module}] %{shortfunc} -> %{level:.4s} %{id:03x}%{color:reset} %{message}"
 
+    # AdditionalParams section for parameters that are passed to chaincode
+    # to specify additional operation modes in which the peer operates.
+    additionalParams:
```

Comment:
Maybe `runtimeParams` would be a better name than `additionalParams` in the config.
I realize the proto is called ChaincodeAdditionalParams, but we can give it a more expressive name in the context of the config.

Reply:
done